rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -113,28 +113,38 @@ Conflicts: rudder-agent
 Requires: fusioninventory-agent fusioninventory-agent-task-inventory
 %endif
 
+## Perl
 %if "%{use_system_perl}" == "true"
-Requires: perl
-%endif
-
-## For Linux
-%if "%{?_os}" != "aix"
-BuildRequires: pam-devel
-%endif
+Requires: perl perl-libwww-perl
+BuildRequires: perl perl-libwww-perl
 
 ## Requirement for cpanminus and fusion
 # rh 6,7
-%if 0%{?rhel} && 0%{?rhel} >= 6
-Requires: perl-IPC-Cmd
-BuildRequires: perl-IPC-Cmd
+%if 0%{?rhel}
+Requires: perl-IPC-Cmd perl-File-Which net-snmp-perl
+BuildRequires: perl-IPC-Cmd perl-File-Which net-snmp-perl
 %endif
 
-# rhel perl core is too minimal, we try to not add too much here
+# rhel 7 perl core is too minimal
 %if 0%{?rhel} && 0%{?rhel} >= 7
-Requires: perl-Digest 
-BuildRequires: perl-Digest
+Requires: perl-Digest perl-DateTime-TimeZone perl-Thread-Queue perl-Data-Dumper
```

Comment:
didn't we said that we were to left the dependencies as they are, and see when it would fail ?